docs/partners/adopt.md

```diff
@@ -0,0 +1,23 @@
+---
+id: adopt 
+title: Adopt-a-Pack 
+---
+
+The Cortex XSOAR Adopt-a-Pack program provides our technical partners a way ‘take over’ a pack that was originally written by Demisto or Palo Alto Networks. The partner becomes the maintainer and supporter of the pack and receives sevral benefits as outlined below.
```

Comment:
```suggestion
The Cortex XSOAR Adopt-a-Pack program provides our technical partners a way to ‘take over’ a pack that was originally written by Cortex XSOAR developers. The partner becomes the maintainer and supporter of the pack and receives several benefits as outlined below.
```

Reply:
I want to leave Demisto in there since the pack may be old.... I'll make other changes.